src/background.js

```diff
@@ -115,7 +128,12 @@ let studyBranch;
 let currentTip = TIPS.NONE;
 
 // Object used to match the user's queries to tips.
-let queryScorer = new QueryScorer();
+let queryScorer = new QueryScorer({
+  variations: new Map([
+    // Recognize "fire fox", "fox fire", and "foxfire" as "firefox".
+    ["firefox", ["fire fox", "fox fire", "foxfire"]],
```

Comment:
is "mozzilla" recognized thanks to levenshtein? That is probably a common mistake for non English mothertongue people (even if they live in the US).

Reply:
Yes, it's an edit distance of 1 away from mozilla, so it's recognized. So is mozila. But mozzila is a distance of 2 away so it wouldn't count... Looking at Verdi's doc, which is based on real user searches as I understand it, there are these variations: "mozila", "mozilla", "mozzila", "mozzilla". mozzila is the only one that's not recognized. So I'll add a variation of "mozila". Between "mozilla" and "mozila" and the edit distance threshold of 1, that will let us catch all these misspellings plus others.